README.md

```diff
@@ -21,7 +21,7 @@ Stable builds are available for:
 - [Windows](doc/DOWNLOADS.md#windows)
 - [macOS](doc/DOWNLOADS.md#macos)
 
-If you want to download unstable builds and help test Nicotine+, see [TESTING.md](doc/TESTING.md).
+If you want to download unstable builds and help test Nicotine+, see [TESTING.md](doc/TESTING.md) and [DEPENDENCIES.md](doc/DEPENDENCIES.md).
```

Comment:
I'd probably remove this and just depend on a link to TESTING.md from the download page.

Reply:
Oh, you mean remove the entire line?

How about move it somewhere into the Getting Involved section?